test/fuzz/gen.ml

```diff
@@ -103,14 +113,17 @@ let expr_available_2_i64 =
 (* let expr_available_3_i64 = [] *)
 
 let expr_available_1_f32 env =
-  [ pair B.funop_32 (const [ S.Nothing ])
-  ; pair B.i32_trunc_f32 (const [ S.Pop; S.Push (Num_type I32) ])
-  ; pair B.i64_trunc_f32 (const [ S.Pop; S.Push (Num_type I64) ])
-  ; pair B.i32_trunc_sat_f32 (const [ S.Pop; S.Push (Num_type I32) ])
-  ; pair B.i64_trunc_sat_f32 (const [ S.Pop; S.Push (Num_type I64) ])
-  ; pair B.f64_promote_f32 (const [ S.Pop; S.Push (Num_type F64) ])
+  [ pair B.f64_promote_f32 (const [ S.Pop; S.Push (Num_type F64) ])
   ; pair B.i32_reinterpret_f32 (const [ S.Pop; S.Push (Num_type I32) ])
+  ; pair B.funop_32 (const [ S.Nothing ])
   ]
+  @ ( if env.Env.conf = Concrete then
+        [ pair B.i32_trunc_f32 (const [ S.Pop; S.Push (Num_type I32) ])
+        ; pair B.i64_trunc_f32 (const [ S.Pop; S.Push (Num_type I64) ])
+        ; pair B.i32_trunc_sat_f32 (const [ S.Pop; S.Push (Num_type I32) ])
+        ; pair B.i64_trunc_sat_f32 (const [ S.Pop; S.Push (Num_type I64) ])
+        ]
+      else [] )
```

Comment:
Could you put all of them as before and let the fuzzer fail if they're not implemented ? We'll implemented them later.